addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -121,6 +121,10 @@ def _create_invoice(self, order, so_line, amount):
                     subtype_id=self.env.ref('mail.mt_note').id)
         return invoice
 
+    @api.model
```

Comment:
note: while the comment might look like requesting changes on things not targeted by the PR, this change request, however, IS important:
```diff
-           invoices.invoice_validate()
-           invoices.action_move_create()
+           invoices.action_invoice_open()  
```